device_name: [24v_payload, LUMA, Regular_Payload]
#Wring PI GPIO number
gpio: [11, 10, 13]

pwm_name: [LUMA_brightness]
pwm_io: [23]

pwm_range: 200
pwm_clock: 100

